Client/Frontend/Browser/URLBarView.swift

```diff
@@ -936,3 +936,50 @@ extension ToolbarTextField: Themeable {
         highlightColor = theme.highlightColor!
     }
 }
+
+class URLBarMinifiedView: BlurWrapper {
```

Comment:
`BlurWrapper` is used as a container containing a UIVisualEffectView and a view that's passed in to blur. It might make more sense to create the `URLBarMinifiedView` as a basic `UIView` subclass and pass it into the constructor like we do for the header/footer view to get the blur - just to keep it more consistent with other uses of `BlurWrapper`.